llvm/docs/TableGen/ProgRef.rst

```diff
@@ -1850,6 +1850,10 @@ and non-0 as true.
 ``!range(``\ *list*\ ``)``
     Equivalent to ``!range(0, !size(list))``.
 
+``!repr(``\ *value*` ``)``
+    Represents *value* as a string. In particular, it allows to print
+    the content of records for debugging purposes.
```

Comment:
AFAICT, we can't do debugging printouts from tablegen yet.
Perhaps rephrase it in a more general terms. E.g. "String format for the value is not guaranteed to be stable. Intended for debugging purposes only."

Reply:
Done, thanks.